package-lock.json

```diff
@@ -3214,12 +3214,61 @@
         "yorkie": "^2.0.0"
       },
       "dependencies": {
+        "@babel/code-frame": {
+          "version": "7.12.13",
+          "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.12.13.tgz",
+          "integrity": "sha512-HV1Cm0Q3ZrpCR93tkWOYiuYIgLxZXZFVG2VgK+MBWjUqZTundupbfx2aXarXuw5Ko5aMcjtJgbSs4vUGBS5v6g==",
+          "dev": true,
+          "optional": true,
+          "requires": {
+            "@babel/highlight": "^7.12.13"
```

Comment:
The changes to package-lock.json seem to be inflating the diff of your PR. I think you can revert? @SamChou19815 can you confirm if that's ok to do?

Reply:
I think it's safe to remove. It's a bug of npm that changes the package lock around with some optional dependencies.